### ISSUE TITLE
Connection closed by remote host popup

### ISSUE THREAD

**Issue by Hakun1n** [mRemoteNG/mRemoteNG#2358]
Whenever session disconnects there is this "Connection closed" popup which originates from PuTTy. That popup steals focus and is annoying when multiple sessions are open which then starts closing after some time.

I am aware that this originates from PuTTy so not really a mRemote issue but could there be some "hacky" way of either suppressing this popup (auto-confirming it silently) or if the "Close window on exit" is enabled on PuTTy then somehow retain the connection tab in mRemote with session history (at least a static snapshot/screenshot) so user can see what was the last action done within that session?

## Expected Behavior
When the session is closed KEEP the tab which session history open (so history is preserverd) but do not throw that popup either.

## Current Behavior
Annoying focus-stealing Popup upon server disconnect. It is even more annoying when multiple-desktops are used. That popup appears on the active desktop even though the mRemote runs on another one.
![image](https://user-images.githubusercontent.com/12685294/220887011-ce5c37e4-e500-4952-b317-a93da64ca447.png)

## Possible Solution
Any way to suppress (force-confirm) this popup from mRemote side?

## Steps to Reproduce (for bugs)
1. Disable "Close window on exit" in PuTTy
2. Connect to remote server and wait for session close from server-side
3. Pop-up appears which steals focus

## Context
- When multiple remote servers are connected (with different connection time) then these popups starts to randomly appear as the sessions are force-closed. This is very annoying and has zero usage as i can clearly see in mRemote that the session got disconnected.
- Enabling "Close window on Exit" is not a solution as that closes the session so user has no clue where the work ended and what has been done on that particular host. This is typical scenario when working on something, then there is some issue which requires google-fu (or any distraction like a call, nature-call, coffe break, etc...) and before the work is restored the session gets disconnected.

## Your Environment

* Version used: 1.76.20.24669
* Operating System and version: Win10 22H2

**Comment by simonai1254** [comment]
The "Close window on Exit" setting in putty sounds like a reasonable thing to do for me as well :)

You may want to change the putty profile used by mRemoteNG (should be something like Settings, External Tools, Launch Putty, don't have it in mind completely). 

What regards that popup, I'm not sure what mRemoteNG can do, even with the putty fork they are using (PuttyNG). Though there are intentions to move away from putty and using a .NET based SSH client (see #181), which would eliminate this problem as well.

**Comment by Hakun1n** [comment]
"The "Close window on Exit" setting in putty sounds like a reasonable thing to do for me as well :)"

It is not because when this is enabled the whole session/tab in mRemote gets closed upon disconnect as well so i have no idea where i ended up and need to dig in logs to see which commands i've already Issued for that host and with what output.

Anyway thanks for mentioning the #181 , i was not aware about that so that might be the ultimate fix for my issue :)

**Comment by simonai1254** [comment]
> "The "Close window on Exit" setting in putty sounds like a reasonable thing to do for me as well :)"
> 
> It is not because when this is enabled the whole session/tab in mRemote gets closed upon disconnect as well so i have no idea where i ended up and need to dig in logs to see which commands i've already Issued for that host and with what output.

You are completely right, read it the wrong way round. One more time where the default is better.

**Comment by Kvarkas** [comment]
ok I got you, will have a look what is posible to do

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `0521ceca` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** Connection closed by remote host popup

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
